exercises/two-fer/two_fer.py

```diff
@@ -1,2 +1,2 @@
-def two_fer(name="you"):
+def two_fer():
```

Comment:
The following would be preferable:

```suggestion
def two_fer(name):
```

Reply:
@cmccandless Would you mind elaborating on why that is preferable?

Reply:
> I have read through the thread: am I understanding that this is preferable because it's only incongruous with the first test rather than all the other tests?

That is correct.